chart/values.yaml

```diff
@@ -619,6 +619,11 @@ workers:
   persistence:
     # Enable persistent volumes
     enabled: true
+    # persistentVolumeClaimRetentionPolicy if statefulset is used, to determine delete when scaled/removed
```

Comment:
Can we add a better comment here? This is not too accurate / is confusing

Reply:
changed, let me know what you this now